packages/vuetify/src/components/VDefaultsProvider/VDefaultsProvider.tsx

```diff
@@ -22,11 +23,13 @@ export const VDefaultsProvider = genericComponent(false)({
   setup (props, { slots }) {
     const { defaults, reset, root, scoped } = toRefs(props)
 
-    provideDefaults(defaults, {
-      reset,
-      root,
-      scoped,
-    })
+    if (!props.disabled) {
+      provideDefaults(defaults, {
+        reset,
+        root,
+        scoped,
+      })
+    }
```

Comment:
If the component starts off with `props.disabled === true` and then later `props.disabled` is set to `false`, then the defaults will still not be provided. It looks like a serious bug to me.

Reply:
The concern is the performance implication for making it reactive and how to address run-time changes because **provideDefaults** can only be run in setup. I talked with @KaelWD and he suggested utilizing effectsScope, which I'm going to look into.